source/quickstart/container.rst

```diff
@@ -6,35 +6,353 @@ Quickstart for Containers
+              You can replace ``${HOME}/minio/data`` with another location in the user's home directory to which the user has read, write, and delete access.
+                 
+         .. tab-item:: Windows
+   
+            .. code-block:: shell
+               :class: copyable
+   
+               docker run \
+                  -p 9000:9000 \
+                  -p 9001:9001 \
+                  --name minio1 \
+                  -e "MINIO_ROOT_USER=ROOTUSER" \
+                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
+                  -v D:\data:/data \
+                  quay.io/minio/minio server /data --console-address ":9001"
+   
+            The example above works this way:
+   
+            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``D:\data``) to the specified virtual container directory (``/data``).
+             - ``-p`` binds a local port to a container port.
+            - ``-v`` sets a file path as a persistent volume location for the container to use.'
+            - ``-e`` sets the environment variables ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``, respectively.
+              Change the example values to use for your container.
+           - When MinIO writes data to ``/data``, that data actually writes to the local path ``D:\data`` where it can persist between container restarts.
+              You can replace ``D:\data`` with another local file location to which the user has read, write, and delete access.
 
-   Contents of Step 2
+#. Connect your browser
 
-#. Step 3
+   Open http://127.0.0.1:9000 in a web browser to access the :ref:`MinIO Console <minio-console>`. 
+   Alternatively, you can enter any of the network addresses specified as part of the container command output.
 
-   Contents of Step 3
+   Access the :ref:`minio-console` by going to a browser and enter one of the Console address specified in the container command's output.
+   For example, :guilabel:`Console: http://192.0.2.10:9001 http://127.0.0.1:9001` in the example output indicates two possible addresses to use for connecting to the Console.
+
+   If you go to port ``9000``, which is used for connecting to the API, you are automatically redirected to the Console port.
+
+   Log in to the Console with the credentials you defined in the ``-e`` environment variables you defined for the container for ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``.
+      In your browser, go to http://127.0.0.1:9000.
+
+   You can use the MinIO Console for general administration tasks like Identity and Access Management, Metrics and Log Monitoring, or Server Configuration. 
+   Each MinIO server includes its own embedded MinIO Console.
+
+   .. image:: /images/minio-console-login.png
+      :width: 400px
+      :alt: MinIO Console displaying login screen
+      :align: center
+
+   From the console, you can create and manage buckets, add objects, and many other tasks.
```

Comment:
```suggestion
   From the Console, you can create and manage buckets, add objects, and many other tasks.
```

Reply:
Actually we already state you can use the Console for all kinds of tasks, so we can just drop this

Reply:
Agree. Moved the login image to right under the Log in line.
Deleted 215.
Moved the the "for more information" to under the bucket image.
Did the same for the MacOS and Windows pages.